packages/core/core/src/PackagerRunner.js

```diff
@@ -130,7 +148,13 @@ export default class PackagerRunner {
       bundle,
       bundleGraph: new BundleGraph(bundleGraph, this.options),
       sourceMapPath: path.basename(bundle.filePath) + '.map',
-      options: this.pluginOptions
+      options: this.pluginOptions,
+      getBundleResult: (bundle: BundleType, bundleGraph: BundleGraphType) => {
```

Comment:
Not sure about this name. I guess `packageBundle` isn't clear that it also includes optimizing... `packageAndOptimizeBundle(bundle)`? `getInlineBundleContents`?

I also think we should only allow inline bundles here, and should throw an error if not. So maybe a name specific to that would be good?

Reply:
I like that idea, updated.